### DISCUSSION TITLE
IBM PC XT Emulator case for lilygo vga32 with FabGL

### DISCUSSION THREAD

**Discussion by fdivitto** [fdivitto/FabGL#179] [Show and tell]
[https://www.thingiverse.com/thing:5028782](https://www.thingiverse.com/thing:5028782)
![IMG_20211023_190320](https://user-images.githubusercontent.com/9754506/138704007-ec89176d-878e-4f5f-ad61-b4c28014b708.jpg)
![IMG_20211023_190327](https://user-images.githubusercontent.com/9754506/138704034-31b092d0-8860-496a-894b-437cbbf9e6f5.jpg)
![IMG_20211023_124258](https://user-images.githubusercontent.com/9754506/138704041-6161c535-9654-4055-beae-66e230fb0785.jpg)

**Comment by lgpf** [comment]
Working on something similar .. bigger :)
Already published some work (empty case) but still pending to publish last one with all guts inside.
Idea is to keep 40% of original size.

https://www.thingiverse.com/thing:6289196
![large_display_24dcb028-a6b2-4dc1-bf58-a7ddf0231844](https://github.com/fdivitto/FabGL/assets/140986386/575a1f54-3980-4fd9-ab50-8b2d97ea7966)

It is almost done with all extensions and leds (but I dont know how to get the hdd led activity so I made a fake seek/blink using attiny)
